content/1.7/back-office/shipping/carriers/Add-edit-carrier.md

```diff
@@ -0,0 +1,214 @@
+
+ - **Tracking URL.** A help text is displayed under the field: "For example: 'http://example.com/track.php?num=@' with '@' where the tracking number should appear.". 
+A tooltip is displayed: "Delivery tracking URL: Type '@' where the tracking number should appear. It will be automatically replaced by the tracking number." The URL should be written as displayed in the tooltip, otherwise, the following error message is displayed: "The url field is invalid.".
+
+## Call to action
+ 
+ - **Previous.** In the first step, this button is disabled.
+ - **Next.** If there is an error, this button is disabled. After clicking on it, the next step is displayed.
+ - **Finish.** If there is an error, this button is disabled. After clicking on it, the carrier is saved and the [carriers listing]({{<ref "carriers-listing.md">}} "Carriers listing") page is displayed with the following success message: "Successful creation" if a carrier was added or "Successful update" if a carrier was edited.
+
+# Multistore (2nd step)
+
+## Call to action
+
+A call to action button at the top right is displayed:
+
+ - Cancel:
+Pressing the button redirects to the [carriers listing]({{<ref "carriers-listing.md">}} "Carriers listing") page.
+
+## Logo
+
+Firstly on the left, the logo is displayed as soon as it's uploaded. Once the logo has been uploaded, a trash icon appears on the right of the logo. When trying to delete the logo, a message is displayed: "Are you sure you want to delete the logo?". Pressing "Ok" will remove the uploaded logo, and the default logo will re-appear in the same place.
+
+## Steps 
+
+ - **1 General settings**
+ - **2 Multistore (If multistore is enabled)**
+ - **3 Shipping locations and costs**
+ - **4 Size, weight, and group access**
+ - **5 Summary**
+
+**This step is displayed only when the multistore is enabled and if there are several shops.**
+
+The shop association block is displayed when the multistore is enabled and if several shops are configured.
+
+👉 See the [specs of the shop association block]({{<ref "../../multistoregeneralspecs.md#shop-association-block">}} "Shop association block") 
+
+## Call to action
+ 
+ - **Previous.** If there is an error, this button is disabled. After clicking on it, the previous step is displayed.
+ - **Next.** If there is an error, this button is disabled. After clicking on it, the next step is displayed.
+- **Finish.** If there is an error, this button is disabled. After clicking on it, the carrier is saved and the [carriers listing]({{<ref "carriers-listing.md">}} "Carriers listing") page is displayed with the following success message: "Successful creation" if a carrier was added or "Successful update" if a carrier was edited.
+
+# Shipping locations and costs (3rd step)
```

Comment:
Both call to action logo and steps are described at the begining. Could you please remove them from this section?